src/web/mjs/connectors/WeebCentral.mjs

```diff
@@ -0,0 +1,84 @@
+import Connector from '../engine/Connector.mjs';
+
+export default class WeebCentral extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'weebcentral';
+        super.label = 'Weeb Central';
+        this.tags = [ 'manga', 'manhua', 'manhwa', 'english' ];
+        this.url = 'https://weebcentral.com';
+    }
+
+    async _getMangas() {
+        const limit = 32;
+        let offset = 0;
+        let mangaList = [];
+        let run = true;
+
+        while (run) {
+            const uri = new URL(`${this.url}/search/data?display_mode=Minimal+Display&limit=${limit}&offset=${offset}`);
+
+            const request = new Request(uri, this.requestOptions);
+            const data = await this.fetchDOM(request, 'article.bg-base-300');
+
+            if (data.length < 1) {
+                run = false;
+                break; // Exit the loop if no more mangas are found
+            }
+
+            const mangas = data.map(element => {
+                const linkElement = element.querySelector('a.link-hover');
+                const title = linkElement.querySelector('h2').textContent.trim();
+                const link = linkElement.href;
+                return {
+                    id: this.getRootRelativeOrAbsoluteLink(link, this.url),
+                    title: title
+                };
+            });
+
+            // Add mangas into mangaList array
+            mangaList = mangaList.concat(mangas);
+
+            // Next page
+            offset += limit;
+        }
+
+        return mangaList;
+    }
+
+    async _getChapters(manga) {
+        const mangaChapterListUrl = manga.id.replace(/\/[^/]+$/, '/full-chapter-list');

```

Comment:
why replace? In case you fear someone pasted a bad link, you just need to control the link format using 

```js
    // May overwrite the default copy & paste matcher in case to support various URL patterns with this connector ...
    canHandleURI(uri) {
        return /https?:\/\/(?:www\.)?template.net/.test(uri.origin);
    }
```

Id from manga list should be fine, providing you used the CSS selector i used in Haruneko.